.github/scripts/run-docker.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+set -euo pipefail
+
+# Parameters
+RUN_SCRIPT=$1
+TODAY=$(date +"%Y%m%d")
+CONFIG_DIR=${HOME}/benchmark-configs
+DATA_DIR=${HOME}/benchmark-results/${TODAY}_gh${GITHUB_RUN_ID}
+
+export CUDA_VISIBLE_DEVICES=0
+
+# Use the latest pytorch image
+TORCH_IMAGE=$(docker images | grep "pytorch-benchmark" | sed -n '1 p')
+TORCH_IMAGE_ID=$(echo $TORCH_IMAGE | tr -s ' ' | cut -d' ' -f3)
+
+echo "Running pytorch-benchmark image ${TORCH_IMAGE_ID}"
+
+mkdir -p ${DATA_DIR}
+
+# Nvidia won't let this run inside docker
+# Make sure the Nvidia GPU is in persistence mode
+sudo nvidia-smi -pm ENABLED -i 0
+# Set the <memory, graphics> clock frequency
+# Need further study on how Nvidia card throttling affect overall performance variance
+sudo nvidia-smi -ac 5001,1500
```

Comment:
just thinking about keeping everything related to tuning in one place- 

is it possible to put all the nvidia, threads, cores, etc settings in the same script? or does it need to be split apart for technical reasons?